warehouse/models/mart/gtfs/fct_daily_scheduled_trips.sql

```diff
@@ -75,7 +75,22 @@ fct_daily_scheduled_trips AS (
         stop_times_grouped.trip_last_arrival_sec,
         stop_times_grouped.service_hours,
         stop_times_grouped.contains_warning_duplicate_primary_key AS contains_warning_duplicate_stop_times_primary_key,
-        stop_times_grouped.contains_warning_missing_foreign_key_stop_id
+        stop_times_grouped.contains_warning_missing_foreign_key_stop_id,
+
+        CASE
+            WHEN (stop_times_grouped.trip_first_departure_sec > 86400) THEN DATE_ADD(service_index.service_date, INTERVAL 1 DAY)
```

Comment:
The logic here I think needs to be more generic -- I think you'd need to do like `CASE WHEN (stop_times_grouped.trip_first_departure_sec / 86400 > 1) THEN  DATE_ADD(service_index.service_date, INTERVAL ROUND(stop_times_grouped.trip_first_departure_sec / 86400) DAY)` -- I think there's no official reason it can't be greater than 1 day (I don't think we have any real examples where it is, but better to be safe than sorry...?)